### ISSUE TITLE
Tooltips for symbols in let/where blocks don't show up

### ISSUE THREAD

**Issue by erykciepiela** [nwolverson/vscode-ide-purescript#162]
For the code in below:
```
i :: Int
i = let k = j in k
  where
    j = 7
```
hovering over `i` displays tooltip `i :: Int` while for `k` and `j` nothing shows up. Tooltips also appear when hovering over other module-local and imported symbols.

PureScript IDE 0.23.2
vsc 1.51.0
purs 0.13.8
pulp 15.0.0

**Comment by nwolverson** [comment]
That is correct, this is common to all PureScript compiler tooling, all type information (tooltips, completion, etc) is only available for top-level identifiers (publicly exported or in the current module after rebuilding).

https://github.com/purescript/purescript/issues/3670
https://github.com/nwolverson/purescript-language-server/issues/109

At such time as `purs ide` might support more information, I'm sure all IDE plugins will do so soon after.
